flux_minimal_inference.py

```diff
@@ -196,17 +194,23 @@ def generate_image(
     t5xxl,
     ae,
     prompt: str,
+    image_path: Optional[str],
     seed: Optional[int],
     image_width: int,
     image_height: int,
     steps: Optional[int],
     guidance: float,
     negative_prompt: Optional[str],
     cfg_scale: float,
+    strength: float,
 ):
     seed = seed if seed is not None else random.randint(0, 2**32 - 1)
     logger.info(f"Seed: {seed}")
 
+    if steps is None:
+        steps = 4 if is_schnell else 50
+    timesteps = get_schedule(steps, image_width * image_height // 64 // 4, shift=not is_schnell)
```

Comment:
Using the schedule parameters from original Flux inference: https://github.com/black-forest-labs/flux/blob/main/demo_st.py#L193